pkg/partmgr/partmgr.go

```diff
@@ -191,14 +213,14 @@ func (p *PartitionManager) ReadAndUpdateSchema() error {
 	schema := config.Schema{}
 	err = json.Unmarshal(resp.Body(), &schema)
 	if err != nil {
-		return errors.Wrap(err, "Failed to Unmarshal schema at path: "+fullPath)
+		return errors.Wrap(err, "Failed to unmarshal schema at path: "+fullPath)
```

Comment:
it's common practice in Go to start messages with lower case.

Reply:
as all msgs are in upper case don't see a point to change this one